.gitignore

```diff
@@ -18,3 +18,4 @@ flake8.report
 junit*.xml
 doc/build
 .cache
+.idea
```

Comment:
What is this `.idea` file ? If it's related to your personal dev environment software, please use a global gitignore.